src/components/container/IntradayHeartRateChart/IntradayHeartRateChart.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Card, DateRangeCoordinator, Layout } from "../../presentational";
+import IntradayHeartRateChart, { IntradayHeartRateChartProps } from "./IntradayHeartRateChart";
+
+
+export default {
+	title: "Container/IntradayHeartRateChart",
+	component: IntradayHeartRateChart,
+	parameters: {
+		layout: 'fullscreen',
+	}
+};
+
+const renderDefault = (args: IntradayHeartRateChartProps) => <Layout colorScheme='auto'>
+	<Card>
+		<IntradayHeartRateChart {...args} />
+	</Card>
+</Layout>;
+
+const renderDRC = (args: IntradayHeartRateChartProps) => <Layout colorScheme='auto'>
+	<Card>
+		<DateRangeCoordinator intervalType={'Day'} initialIntervalStart={new Date()}>
+			<IntradayHeartRateChart {...args} />
+		</DateRangeCoordinator>
+	</Card>
+</Layout>;
+
+export const Default = {
+	render: renderDefault,
+	args: {
+		previewState: "Default",
+		aggregationOption: "avg"
```

Comment:
I'm unclear what the different aggregation options control and when it would appropriate to use each of them.  Is there some more information we can add?

Reply:
@brandoncharnesky More detail on the aggregation options can be found [here](https://developer.mydatahelps.org/api/device_data_v2.html)

To your point and offhand, I don't have a good example of when to use max/min on this particular graph. I kept it as an enum for flexibility but can restrict to avg.  @chrisnowak Do you have any thoughts on supporting all the aggregate functions vs limiting to avg?